extensions/markdown/src/tableOfContentsProvider.ts

```diff
@@ -65,14 +68,24 @@ export class TableOfContentsProvider {
 		return toc;
 	}
 
+	private static getHeaderLevel(markup: string): number {
+		if (markup == '=') {
```

Comment:
These two comparisons need to use `===`. I think this is what is causing the Travis build to fail